types/slate/index.d.ts

```diff
@@ -1131,12 +1131,12 @@ export interface Plugin {
     schema?: SchemaProperties;
 }
 
-export type Plugins = Plugin | Plugin[] | PluginStack;
-export interface PluginStack extends Array<Plugins> {}
+export type Plugins = Plugin | NestedPlugins;
+export interface NestedPlugins extends Array<Plugins> {}
 
 export interface EditorProperties {
     onChange?: (change: { operations: Immutable.List<Operation>; value: Value }) => void;
-    plugins?: PluginStack;
+    plugins?: Plugins;
```

Comment:
I think you can pass a plain Plugin, so
```ts
plugins?: Plugin | Plugins;
```
is correct?

Reply:
> I think you can pass a plain Plugin

I really think you can't. 

Slate's `CorePlugin` most definitely expects a plugin array at `plugins`. Here's some pointers.
https://github.com/ianstormtaylor/slate/blob/master/packages/slate/src/plugins/core.js#L20  
https://github.com/ianstormtaylor/slate/blob/master/packages/slate/src/plugins/core.js#L200 

The «act as one plugin»-case is covered by the fact that the editor controller/component in `slate-react` accepts all properties  of a regular plugin. A case that @types/slate-react already tests. 
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/slate-react/slate-react-tests.tsx#L49-L56

Reply:
Hey, you are right! Thanks for letting me know!